### PR TITLE
[App Configuration]Add sample for creating feature flag

### PR DESCRIPTION
**Pull request by ZhijunZhao**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@ZhijunZhao - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by ZhijunZhao** [comment]
@zhenlan for awareness.

File: 101-app-configuration-store-ff/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "configStoreName": {
+      "type": "string",
+      "metadata": {
```
Comment (bmoore-msft) [inline 101-app-configuration-store-ff/azuredeploy.json:7]: Add a globally unique defaultValue using uniqueString()

File: 101-app-configuration-store-ff/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      "name": "[parameters('configStoreName')]",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "standard"
+      },
+      "resources": [
+        {
+          "type": "keyValues",
+          "apiVersion": "2020-07-01-preview",
+          "name": "[concat('.appconfig.featureflag~2F', parameters('featureFlagKey'), '$', parameters('featureFlagLabel'))]",
```
Comment (bmoore-msft) [inline 101-app-configuration-store-ff/azuredeploy.json:54]: if this naming pattern has a specific purpose, explain in the readme... if not, simplify.

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
Also see the pipeline results for some changes that need to be made.

**Comment by bmoore-msft** [comment]
@ZhijunZhao - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

File: 101-app-configuration-store-ff/azuredeploy.json (hunk 2 above)
Comment (ZhijunZhao) [reply 101-app-configuration-store-ff/azuredeploy.json:54]: Thanks for the feedback. Explanation is added.

**Comment by bmoore-msft** [comment]
@ZhijunZhao - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by bmoore-msft** [comment]
@ZhijunZhao - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by ZhijunZhao** [comment]
@bmoore-msft Could you help to check below error message? I don't know how to fix it. 

```
New-AzResourceGroupDeployment : 9:52:47 AM - The deployment 'mainTemplate-deployment' failed with error(s). Showing 1 out of 1 error(s).
Status Message: The client '5991c0de-1362-4164-8ac0-0ab3b91175ee' with object id '5991c0de-1362-4164-8ac0-0ab3b91175ee' does not have authorization to perform action 'Microsoft.AppConfiguration/configurationStores/write' over scope '/subscriptions/0cec7090-2e08-4498-9337-eb96ade50821/resourcegroups/azdo-7e7230be-b0bd-4987-b2a1-33b5d30f8dc2/providers/Microsoft.AppConfiguration/configurationStores/cia20c0e236ded4f64' or the scope is invalid. If access was recently granted, please refresh your credentials. (Code:AuthorizationFailed)
```

**Comment by bmoore-msft** [comment]
@ZhijunZhao - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by ZhijunZhao** [comment]
> @bmoore-msft Could you help to check below error message? I don't know how to fix it.
> 
> ```
> New-AzResourceGroupDeployment : 9:52:47 AM - The deployment 'mainTemplate-deployment' failed with error(s). Showing 1 out of 1 error(s).
> Status Message: The client '5991c0de-1362-4164-8ac0-0ab3b91175ee' with object id '5991c0de-1362-4164-8ac0-0ab3b91175ee' does not have authorization to perform action 'Microsoft.AppConfiguration/configurationStores/write' over scope '/subscriptions/0cec7090-2e08-4498-9337-eb96ade50821/resourcegroups/azdo-7e7230be-b0bd-4987-b2a1-33b5d30f8dc2/providers/Microsoft.AppConfiguration/configurationStores/cia20c0e236ded4f64' or the scope is invalid. If access was recently granted, please refresh your credentials. (Code:AuthorizationFailed)
> ```

I closed and reopened the PR. Seems like the error is gone.

**Comment by bmoore-msft** [comment]
@ZhijunZhao - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Pull request merged**